design-patterns/gc-intro/index.json-ld

```diff
@@ -0,0 +1,224 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Introduction block - Documentation and working example",
+		"fr": "Bloc d’introduction - Documentation et example pratique"
+	},
+	"description": {
+		"en": "Guidance about using the introduction block pattern on Canada.ca.",
+		"fr": "Orientation sur l’utilisation de la configuration de conception du bloc d’introduction sur Canada.ca."
+	},
+	"modified": "2023-08-16",
+	"componentName": "gc-intro",
+	"status": "stable",
+	"version": "1.0",
```

Comment:
There is no stability indicator and no version because a design pattern out of scope of our versioning api

```suggestion
```